CodeStarterCamp_Week2/checkPassedLuckyNumbers.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  checkPassedLuckyNumbers.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by minsong kim on 2023/03/01.
+//
+
+import Foundation
+var passedLuckyNumbers: Dictionary<String, Set> = [String: Set<Int>]()
+
+func saveLuckyNumbers(time: Int) {
+    for count in 1...time {
+        pickNumbers()
+        passedLuckyNumbers["\(count)회차"] = luckyNumbers
+    }
+}
+
+func getPassedLuckyNumbers(time: Int) {
+    if let numbers = passedLuckyNumbers["\(time)회차"] {
+        print("\(time)회차의 로또 당첨 번호는 ", terminator: "")
+        print(numbers.map{ (number: Int) -> String in return String(number)}.joined(separator:", "), terminator: " 입니다.")
```

Comment:
map함수를 사용해 보셨군요!
다른 방법을 사용해보신점 좋은 방향성이라 생각합니다! 다만, 이러한 고차함수의 단점은 코드를 읽기 힘들수 있다는 점이죠..!
필요하다면 개행을 통해 가독성을 조금 챙길 수 있을거 같아요!
그리고 일반적으로 이렇게 개행하기도 합니다.
```suggestion
        print(numbers.map{ (number: Int) -> String in 
            return String(number)
        }
        .joined(separator:", "), terminator: " 입니다.")
```